ImportError for check_control_in_treatment

After #97 was merged to master, tests are failing with `ImportError` for `check_control_in_treatment` due to merge conflict as follows:
```
ImportError while loading conftest '/Users/jeong/development/causalml/tests/conftest.py'.
tests/conftest.py:4: in <module>
    from causalml.dataset import synthetic_data
causalml/dataset/__init__.py:8: in <module>
    from .synthetic import get_synthetic_preds, get_synthetic_preds_holdout
causalml/dataset/synthetic.py:15: in <module>
    from causalml.inference.meta import BaseXRegressor, BaseRRegressor, BaseSRegressor, BaseTRegressor
causalml/inference/meta/__init__.py:1: in <module>
    from .slearner import LRSRegressor, BaseSLearner, BaseSRegressor, BaseSClassifier
causalml/inference/meta/slearner.py:16: in <module>
    from causalml.inference.meta.utils import check_control_in_treatment, convert_pd_to_np
E   ImportError: cannot import name 'check_control_in_treatment'
```

--- a/causalml/inference/meta/rlearner.py
+++ b/causalml/inference/meta/rlearner.py
@@ -1,10 +1,5 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from future.builtins import super
 from copy import deepcopy
 import logging
-import pandas as pd
 import numpy as np
 from tqdm import tqdm
 from scipy.stats import norm

--- a/causalml/inference/meta/slearner.py
+++ b/causalml/inference/meta/slearner.py
@@ -1,7 +1,3 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from future.builtins import super
 import logging
 import numpy as np
 from tqdm import tqdm
@@ -13,7 +9,7 @@ from copy import deepcopy
 from causalml.inference.meta.explainer import Explainer
 from causalml.inference.meta.utils import check_treatment_vector
 from causalml.metrics import regression_metrics, classification_metrics
-from causalml.inference.meta.utils import check_control_in_treatment, convert_pd_to_np
+from causalml.inference.meta.utils import convert_pd_to_np
 
 
 logger = logging.getLogger('causalml')

--- a/causalml/inference/meta/tlearner.py
+++ b/causalml/inference/meta/tlearner.py
@@ -1,7 +1,3 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from future.builtins import super
 from copy import deepcopy
 import logging
 import numpy as np
@@ -11,12 +7,11 @@ from sklearn.exceptions import ConvergenceWarning
 from sklearn.neural_network import MLPRegressor
 from sklearn.utils.testing import ignore_warnings
 from xgboost import XGBRegressor
-import shap
 
 from causalml.inference.meta.explainer import Explainer
 from causalml.inference.meta.utils import check_treatment_vector
 from causalml.metrics import regression_metrics, classification_metrics
-from causalml.inference.meta.utils import check_control_in_treatment, convert_pd_to_np
+from causalml.inference.meta.utils import convert_pd_to_np
 
 
 logger = logging.getLogger('causalml')

--- a/causalml/inference/meta/tmle.py
+++ b/causalml/inference/meta/tmle.py
@@ -1,5 +1,4 @@
 import logging
-import pandas as pd
 import numpy as np
 from scipy.optimize import minimize
 from scipy.special import expit, logit

--- a/causalml/inference/meta/xlearner.py
+++ b/causalml/inference/meta/xlearner.py
@@ -1,10 +1,6 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
 from future.builtins import super
 from copy import deepcopy
 import logging
-import pandas as pd
 import numpy as np
 from tqdm import tqdm
 from scipy.stats import norm

--- a/requirements-test.txt
+++ b/requirements-test.txt
@@ -1,1 +1,1 @@
-pytest==4.6
+pytest

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,7 +9,6 @@ statsmodels>=0.9.0
 seaborn
 Cython
 xgboost
-future
 pydotplus
 tqdm
 eli5

